lua/entities/gmod_wire_expression2/core/player.lua

```diff
@@ -325,6 +325,26 @@ e2function number entity:isTyping()
 	return plys[this] and 1 or 0
 end
 
+-- isCoding
+local editing = {}
+
+local function handleEditUpdate(ply,state)
+	editing[ply] = (state and true or nil)
+	print("Handled edit update: " .. tostring(state))
+end
+ 
+hook.Add("IsEditing","IsEditing_Hook",function(ply,set) handleEditUpdate(ply,set) end)
+hook.Add("PlayerDisconnected","IsEditing_Disconnected",function(ply) editing[ply] = nil end)
+net.Receive("IsEditing", function(netlen,ply) handleEditUpdate(ply,net.ReadBool()) end)
+ 
+__e2setcost(5)
+e2function number entity:isEditing()
+	if !IsValid(this) then return -1 end
+	if !this:IsPlayer() then return -1 end
```

Comment:
I'll change it, because it makes sense to have everything work the same way.
However, I'd like to comment that there should be *some* way for someone to tell that the command has errored. Since this function returns a 0 or a 1, returning a 0 on error doesn't tell you much...